File: backend/src/main/java/com/example/masters/dto/device/DeviceSimpleResponse.java
package com.example.masters.dto.device;

import lombok.Builder;
import lombok.Data;

import java.util.UUID;

@Data
@Builder
public class DeviceSimpleResponse {
    private UUID id;
    private String title;
    private String description;
    private String inventoryNumber;
    private String type;
}

File: frontend/app/src/main/java/com/example/smartlab/model/DeviceSimpleResponse.kt
package com.example.smartlab.model

import android.os.Parcelable
import kotlinx.android.parcel.Parcelize

@Parcelize
data class DeviceSimpleResponse(
    val id: String,
    val title: String?,
    val description: String?,
    val inventoryNumber: String?,
    val type: String?
): Parcelable